networkx/algorithms/cycles.py

```diff
@@ -106,9 +106,8 @@ def simple_cycles(G):
     """Find simple cycles (elementary circuits) of a directed graph.
 
     An simple cycle, or elementary circuit, is a closed path where no
```

Comment:
Typo here: An -> A. Also should italicize definitions *simple cycle* and *elementary circuit*.